source/_docs/style-guide.md

```diff
@@ -469,6 +469,35 @@ Use panels for extraneous but useful information such as troubleshooting section
 ```
 </div>
 
+## Tables
+
+Markdown tables are defined like this:
+
+<div class="style-example" markdown="1">
+| Table Header 1     | Table Header 2 |
+| ------------------ | :------------: |
+| Table Data 1.1     | Table Data 2.1 |
+| Table Data 1.2     | Table Data 2.2 |
+| Table Data 1.3     | Table Data 2.3 |
+<hr class="source-code">
+```markdown
+| Table Header 1     | Table Header 2 |
+| ------------------ | -------------- |
+| Table Data 1.1     | Table Data 2.1 |
+| Table Data 1.2     | Table Data 2.2 |
+| Table Data 1.3     | Table Data 2.3 |
+```
+</div>
+
+## Tooltips
```

Comment:
Can you make the same adjustments to the tooltips section? 
- Improved inro that describes the intended usage. (doesn't need to mention alternatives specifically, but after reading it should be clear when to use tooltips vs something else like a definition list)
- Align the style of the example with the rest of the page - [refer to this commit for example](https://github.com/pantheon-systems/documentation/pull/3457/commits/434c4dfb697978fdaa831e053299f69af15610ba#diff-135923fe09a22b7615f023b1270a8a6e).


Also, I suggest replacing the example tooltip here with an example showing how to include links. Such as [the tooltip here](https://pantheon.io/docs/guides/multisite/considerations/): 
![screen shot 2018-03-13 at 1 26 10 pm](https://user-images.githubusercontent.com/10119525/37362838-59d2e1c2-26c4-11e8-84cb-0e78dc8fb9d8.png)